src/lib/twizzler-abi/src/kso.rs

```diff
@@ -115,13 +115,16 @@ pub enum KactionError {
     InvalidArgument = 1,
     /// The object was not found.
     NotFound = 2,
+    /// Out of resources.
+    OutOfResources = 3,
 }
 
 impl From<u64> for KactionError {
     fn from(x: u64) -> Self {
         match x {
             1 => KactionError::InvalidArgument,
             2 => KactionError::NotFound,
+            3 => KactionError::OutOfResources,
```

Comment:
I both like this name but am also confused by it.  OOR could be memory or could be CPU (back in the old days anyway) so maybe you need to be more specific?

Reply:
Ah, this is currently used to indicate that allocating an interrupt failed. I had intended this to be a more generic statement of "you called a Kaction that asked for a resource, and there aren't any left." We could add an explicit OOM variant too, if that would help clarify.

Reply:
Yeah. Something to clear it up.